api/static/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>openyieldtables API</title>
+    <style>
+      body {
+        font-family: "Courier New", Courier, monospace;
+        margin: 10vh 10vw;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Welcome to the openyieldtables API</h1>
+    <br />
+    <h4>This is the API for the openyieldtables project.</h4>
+    <p>
+      The documentation of the project can be found
+      <a href="https://openyieldtables.readthedocs.io">here</a>. The API
+      documentation can be found <a href="/docs">here</a>.
+    </p>
+    <br />
+    <h4>The following endpoints are available:</h4>
+    <p>
+      1. GET: https://yieldtables.org/v1/yield-tables-meta<br />
+      2. GET: https://yieldtables.org/v1/yield-tables-meta/:id<br />
+      3. GET: https://yieldtables.org/v1/yield-tables/:id<br />
+    </p>
+  </body>
```

Comment:
I would not document them here since they are documented in the docs and here it's just a maintenance effort.